### PR TITLE
[CUBRIDQA-950]add -n option to runone.sh

### PR DESCRIPTION
**Pull request by jinmeihua**
http://jira.cubrid.org/browse/CUBRIDQA-950

### CODE REVIEW

**Comment by eseokoh** [comment]
Any reason to hard code the db name?
```
db=ctldb
```

**Comment by jinmeihua** [comment]
No, it was just from the first version of ctltool/runone.sh.
https://oss.navercorp.com/CUBRID/qatools/blob/55254c619fd5da9e4237b6a07ab652c83e41dde6/ctltool/runone.sh

**Comment by jinmeihua** [comment]
@fanzaiqiang 
QA issue: http://jira.cubrid.org/browse/CUBRIDQA-950

> No problem for changes.
> Please open a QA task issue for eliminating hard codes on db name.

**Comment by jinmeihua** [comment]
commit 202e2b7:

- Let `ctp.sh isolation ` to support `backup_core_file_yn` parameter. Default value is yes.

So that we can use
 ```
config="isolation.config"
echo "scenario=`pwd`" > $config
echo "backup_core_file_yn=no" >> $config
ctp.sh isolation -c $config > ctp.log 2>&1 
```
instead of 
```
CTP_HOME=`cd $(dirname $(readlink -f $init_path))/..; pwd`
sh $CTP_HOME/isolation/ctltool/runone.sh -n cbrd_23039.ctl 300 
```


- Also add -n option to runall.sh too.

File: CTP/isolation/src/com/navercorp/cubridqa/isolation/Context.java [hunk 1]
```diff
@@ -95,6 +97,7 @@ public void reload() throws IOException {
 		this.shouldUpdateTestCase = CommonUtils.convertBoolean(getProperty(ConfigParameterConstants.TESTCASE_UPDATE_YES_OR_NO, "false")) && !CommonUtils.isEmpty(getTestCaseBranch());
 		this.isWindows = getTestPlatform().equalsIgnoreCase("windows");
 		this.isContinueMode = CommonUtils.convertBoolean(getProperty(ConfigParameterConstants.TEST_CONTINUE_YES_OR_NO, "false"));
+		this.isContinueMode = CommonUtils.convertBoolean(getProperty(ConfigParameterConstants.TEST_CONTINUE_YES_OR_NO, "false"));
```
Comment (fanzaiqiang) [inline CTP/isolation/src/com/navercorp/cubridqa/isolation/Context.java]: duplication:
```
this.isContinueMode= ...
```

File: CTP/isolation/src/com/navercorp/cubridqa/isolation/Context.java [hunk 2]
```diff
@@ -177,6 +180,10 @@ public boolean isWindows() {
 		return this.isWindows;
 	}
 
+	public boolean getBackupCoreFile() {
```
Comment (fanzaiqiang) [inline CTP/isolation/src/com/navercorp/cubridqa/isolation/Context.java]: Rename:
getBackupCoreFile => isBackupCoreFile

File: CTP/isolation/src/com/navercorp/cubridqa/isolation/Context.java [hunk 3]
```diff
@@ -67,6 +67,8 @@
 	private boolean reInstallTestBuildYn = false;
 	private String scenario;
 	private boolean isExecuteAtLocal = false;
+	private boolean backupCoreFile = true;
```
Comment (fanzaiqiang) [inline CTP/isolation/src/com/navercorp/cubridqa/isolation/Context.java]: Not used?

**Pull request merged**